projects/packages/my-jetpack/_inc/components/card/style.module.scss

```diff
@@ -0,0 +1,50 @@
+.container {
+	padding: calc( var( --spacing-base ) * 3 ); // 24px
+	background: var( --jp-white );
+	border-radius: var( --jp-border-radius );
+	box-shadow: 0 0 40px rgba(0, 0, 0, 0.08);
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	box-shadow:
+		0 0 0 1px var( --jp-gray-10 ) inset,
```

Comment:
Nitpick: It may be something from storybook, but it looks like it's overriding this style with a 1.5px box shadow. Coming from an `is-secondary` class.

However, this is also how the buttons are currently styled in My Jetpack - the comps are different than the existing buttons we have on other cards.

![Screenshot 2023-09-26 at 10 05 42 AM](https://github.com/Automattic/jetpack/assets/18016357/1302a663-989c-4495-bac7-3b3b8d3e4016)

Reply:
I can update the style of the button, but since this is going to be displayed on the same screen as the existing buttons, I'd rather leave it like this (at least for now).